### PR TITLE
CC-25347: publisher trigger plugins integration guide

### PR DESCRIPTION
**Pull request by artem-png**
## PR Description

Epics - https://spryker.atlassian.net/browse/CC-20636, https://spryker.atlassian.net/browse/CC-22522
IG tickets - https://spryker.atlassian.net/browse/CC-25347,  https://spryker.atlassian.net/browse/CC-25348

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Comment by dmiseev** [comment]
Ready for PO review.

**Review by dmiseev — APPROVED** [review]

**Review by KarinOf — DISMISSED** [review]

File: docs/marketplace/dev/feature-integration-guides/202304.0/marketplace-product-offer-feature-integration.md [hunk 1]
```diff
@@ -0,0 +1,1763 @@
...
+            new MerchantProductOfferSearchPublisherTriggerPlugin(),
+            new ProductOfferPublisherTriggerPlugin(),
+        ];
+    }       
+}
+```
+</details>
+
+#### Set up event listeners
+
```
Comment (VadymSachenko) [inline docs/marketplace/dev/feature-integration-guides/202304.0/marketplace-product-offer-feature-integration.md:249]: Add an introductory phrase. Other IGs have the following intro phrase for this step:
```suggestion
{% info_block infoBox %}

In this step, you enable publishing of table changes—create, edit, delete to `spy_category_image_storage` and synchronization of data to Storage.

{% endinfo_block %}
```

File: docs/marketplace/dev/feature-integration-guides/202304.0/marketplace-product-offer-prices-feature-integration.md [hunk 2]
```diff
@@ -0,0 +1,1185 @@
...
+Generate a new translation cache for Zed:
+
+```bash
+console translator:generate-cache
+```
+
+### 4) Configure export to Redis
+
+Set up event listeners
+
```
Comment (VadymSachenko) [inline docs/marketplace/dev/feature-integration-guides/202304.0/marketplace-product-offer-prices-feature-integration.md:123]: Add an introductory phrase. Other IGs have the following intro phrase for this step. Maybe it fits here as well:
```suggestion
{% info_block infoBox %}

In this step, you enable publishing of table changes—create, edit, delete to `spy_category_image_storage` and synchronization of data to Storage.


{% endinfo_block %}
```

File: docs/marketplace/dev/feature-integration-guides/202304.0/marketplace-product-offer-feature-integration.md (hunk 1 above)
Comment (artem-png) [reply docs/marketplace/dev/feature-integration-guides/202304.0/marketplace-product-offer-feature-integration.md:248]: Let's not add it, as product offer entity doesn't have it's own *_search table. It updates other search indexes

Comment (VadymSachenko) [reply docs/marketplace/dev/feature-integration-guides/202304.0/marketplace-product-offer-feature-integration.md:248]: @artem-png anyway, we need an introductory sentence here. Wee need to explain the step. For example, "Register the following plugins" or anything alike. We can't leave blank.

Comment (dmiseev) [reply docs/marketplace/dev/feature-integration-guides/202304.0/marketplace-product-offer-feature-integration.md:248]: @artem-png I see 2 options here:
- You could mention 2 product search tables (abstract and concrete).
- You could add a small description that such plugins will expand product search tables with offers.

**Review by dmiseev — APPROVED** [review]

File: _includes/pbc/all/install-features/202304.0/install-the-category-image-feature.md [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+
+Make sure that when a category image is created, updated or deleted, it is exported or removed from Redis accordingly.
+
+{% endinfo_block %}
+
+| STORAGE TYPE | TARGET ENTITY | EXAMPLE EXPECTED DATA IDENTIFIER |
+| --- | --- | --- |
+| Redis | Category Image | `kv:category_image:en_us:1` |
+
+**Example expected data fragment**
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-category-image-feature.md:276]:
```suggestion
**An example example of expected data fragment**
```

**Review by VadymSachenko — CHANGES_REQUESTED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**